app/clubs/templates/clubs/available-clubs.html

```diff
@@ -1,8 +1,475 @@
-{% extends 'base.html' %}
+<!-- {% extends 'base.html' %}
 {% block body %}
 
 <section>
   <div>Join Clubs</div>
 </section>
 
-{% endblock %}
\ No newline at end of file
+{% endblock %} -->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
If you want to use more Django-specific tools, try putting the html inside the base Django template located in app/core/templates/base.html, it allows you to write less boiler plate. I also encourage you to look up how static files work in django, it can be a bit confusing, but it allows you to offload the css/js files to a S3 bucket in production to improve performance. 

Here's an example of the template system and loading a static css file:

```html
{% extends 'base.html' %}
{% block head %}
{% load static %}
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" />
<link rel="stylesheet" href="{% static 'clubs/available-clubs.css' %}">
{% endblock %}
{% block body %}
<div class="container">
<!-- etc... -->
</div>
{% endblock %}
```

Here's some resources talking about both:
- Static files: <https://docs.djangoproject.com/en/5.1/howto/static-files/>
- Templates: <https://docs.djangoproject.com/en/5.1/topics/templates/>
- Using template variables, etc: <https://www.geeksforgeeks.org/django-templates/>
- Template blocks: <https://www.w3schools.com/django/ref_tags_block.php>